pywbem/_logging.py

```diff
@@ -117,279 +167,183 @@
 DEFAULT_LOG_DESTINATION = 'none'
 
 
-class MetaPywbemLoggers(type):
-    """
-    **Experimental:** *New in pywbem 0.11 as experimental.*
-
-    This metaclass allows the definition of __str__ and __repr__ at the
-    class level in the subclass.
-    """
-    loggers = {}
-
-    def __str__(cls):
-        return 'PywbemLoggers(loggers={s.loggers!r})'.format(s=cls)
-
-    def __repr__(cls):
-        return 'PywbemLoggers({s.loggers!r})'.format(s=cls)
-
-
-# pylint: disable=no-init
-class PywbemLoggers(six.with_metaclass(MetaPywbemLoggers)):
+def define_loggers_from_string(log_spec_str, log_filename=None):
+    # pylint: disable=line-too-long
     """
-    **Experimental:** *New in pywbem 0.11 as experimental.*
+    Create the pywbem loggers defined by the input string in the following
```

Comment:
These functions are not "creating" or "defining" the loggers, but are "configuring" them or "setting them up". I suggest that we choose some wording of that sort, and stick with it consistently, both in the function names and in the description.

Reply:
When we move these we will rename them.